x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/lucene/LuceneOperator.java

```diff
@@ -231,12 +232,20 @@ int position() {
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append(this.getClass().getSimpleName()).append("[");
-        sb.append("maxPageSize = ").append(maxPageSize);
+        sb.append("shards = ").append(sortedUnion(processedShards, sliceQueue.remainingShards()));
```

Comment:
I wonder if it is also worth including it in the status object as well?

Reply:
What about having it in the status only, instead of the toString? I imagine toString is a bit like an identity. So the decision would be whether we view this is part of the identify?

Reply:
I believe it should be. Each LuceneOperator is only used with list of shards it is originally initialized with